src/views/RecIM/components/Forms/InviteParticipantForm/index.js

```diff
@@ -0,0 +1,97 @@
+import { Grid } from '@mui/material';
+import { useState, useEffect } from 'react';
+import GordonDialogBox from 'components/GordonDialogBox';
+import { ParticipantList } from './../../List';
+import GordonQuickSearch from 'components/Header/components/QuickSearch';
+import GordonSnackbar from 'components/Snackbar';
+import { addParticipantToTeam } from 'services/recim/team';
+import { useCallback } from 'react';
+
+const InviteParticipantForm = ({
+  closeWithSnackbar,
+  openInviteParticipantForm,
+  setOpenInviteParticipantForm,
+  teamID,
+}) => {
+  const [disableUpdateButton, setDisableUpdateButton] = useState(true);
+  const [inviteList, setInviteList] = useState([]);
+  const [snackbar, setSnackbar] = useState({ message: '', severity: null, open: false });
+
+  const createSnackbar = useCallback((message, severity) => {
+    setSnackbar({ message, severity, open: true });
+  }, []);
+
+  useEffect(() => {
+    setDisableUpdateButton(!inviteList || !inviteList.length);
+  }, [inviteList]);
+
+  const onSearchSubmit = (username) => {
+    // Check if participant exists in invite list
+    if (inviteList.includes({ Username: username })) {
+      createSnackbar('Participant already in list', 'error');
+      return;
+    }
+
+    setInviteList([...inviteList, { Username: username }]);
+  };
+
+  const removeInvite = (username) => {
+    setInviteList(inviteList.filter((participant) => participant.Username !== username));
+  };
+
+  const handleSubmit = () => {
+    inviteList.forEach((value) => {
+      let participantData = {
+        Username: value.Username,
+        RoleTypeID: 2,
+      };
+      addParticipantToTeam(teamID, participantData).catch((reason) => {
+        console.log(reason);
```

Comment:
console log

Reply:
This is actually a placeholder, and does not necessarily need to be removed as it is catching the error the backend throws. Ultimately we should be adding snackbar notifications whenever an operation does not succeed. This is what that is in place of. I can remove it though if we would like to go back through later and add more in depth error handling.

Reply:
i see, as long as there is a purpose then it's alright. however, errors that get thrown by the backend would appear on the console log regardless of catch.